tests/functional/odf-cli/test_mclock_iops_override.py

```
@@ -0,0 +1,269 @@
+        self.odf_cli.wait_for_ceph_config_value(
+            f"osd.{target_osd}",
+            self.config_key,
+            self.custom_iops_value,
+        )
+
+        self.odf_cli.run_ceph_config_rm(f"osd.{target_osd}", self.config_key)
+        log.info("Removed override for osd.%s", target_osd)
+
+        value_after = self.odf_cli.run_ceph_config_get("osd", self.config_key)
+        assert float(value_after) == self.default_value, (
+            f"After removal expected {self.default_value}, " f"got {value_after}"
```

Ruff (0.16.1)

[warning] Implicitly concatenated string literals on one line

Combine string literals

(ISC001)